README.md

```diff
@@ -12,6 +12,12 @@ This style guide is a compilation of [atomic components](http://bradfrost.com/bl
 - Link to the production files at `styleguide/public/assets/`
 
 ## To begin working:
+**Environment setup (mac)**
+
+ - Have [`homebrew`](https://brew.sh/) installed
+ - `brew install nvm`
```

Comment:
I'm having trouble with this... there're some things `brew` tells you to do after install but it doesn't seem to be super straightforward.